### PR TITLE
[INTERNAL] server: Expose serveIndex options

### PR DESCRIPTION
**Pull request by jkoenig134**
In the last PR (#253) i forgot to expose the cli-arguments to the serveIndex Middleware.
Here is the solution ...

## Pull Request Checklist
- [x] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#-contributing-code)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#how-to-contribute) section 
- [x] [No merge commits](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#no-merge-commits)
- [x] [Correct commit message style](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

**Comment by jkoenig134** [comment]
@RandomByte first try 🎉 all pipelines passed!

File: lib/server.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ module.exports = {
 	 */
 	async serve(tree, {
 		port: requestedPort, changePortIfInUse = false, h2 = false, key, cert,
-		acceptRemoteConnections = false, sendSAPTargetCSP = false}) {
+		acceptRemoteConnections = false, sendSAPTargetCSP = false, showHidden = false, showDetails = false
```
Comment (RandomByte) [inline lib/server.js]: In https://github.com/SAP/ui5-server/pull/254 we decided on having defaults `showHidden = true` and `showDetails = true` to stay compatible with the current serve index behavior.

**Comment by jkoenig134** [comment]
Hey @RandomByte, i will fix all the commented things on cli and server soon.

**Comment by jkoenig134** [comment]
@RandomByte already pushed commit messages can only be changed ba hard reverting. You could squash merge the pull request.
Default values are changed.

**Pull request merged**